docs/self-managed/operational-guides/update-guide/820-to-830.md

```diff
@@ -8,6 +8,12 @@ description: "Review which adjustments must be made to migrate from Camunda 8.2.
 
 The following sections explain which adjustments must be made to migrate from Camunda 8.2.x to 8.3.0 for each component.
 
+:::caution
+Updating Operate, Tasklist and Optimize from 8.2.x to 8.3.0 will potentially take longer than expected, depending on the data to be migrated.
+Additionally, we identified some bugs that could also prevent the migration from succeeding. These are being addressed and will be available in an upcoming 8.3.1 patch. We suggest not updating until the patch is released.
+After the patch, migration will succeed, but will still take long dependent on data to be migrated.
```

Comment:
I think we could skip that part, we indicate it takes a long time generally + hint on known issues to be resolved, thank you!
```suggestion
```